paris/src/testDebug/java/com/airbnb/paris/proxies/TextViewStyleApplierTest.kt

```diff
@@ -54,6 +56,19 @@ class TextViewStyleApplierTest {
         assertFalse(view.compoundDrawables[0].bounds.isEmpty)
     }
 
+    @Test
+    fun fontFamily_sansSerif() {
+        applier.apply(R.style.Test_TextViewStyleApplier_FontFamily_String)
+        assertEquals(Typeface.create("sans-serif", Typeface.NORMAL), view.typeface)
```

Comment:
Same comment as above, isn't this the default? If so we should set the typeface to something else on the view at the beginning of the test.